### PR TITLE
docs(react-avatar): Adding more AvatarGroup stories

### PR DESCRIPTION
**Pull request by sopranopillow**
## Current Behavior

AvatarGroup doesn't cover good examples.

## New Behavior

AvatarGroup now has good examples.

## Related Issue(s)

#22240

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `45.538 kB`<br />`13.298 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `184.781 kB`<br />`51.176 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `33.988 kB`<br />`11.108 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/0d4794d3924d648c788993e087a48dd45fe10a1b'>0d4794d3924d648c788993e087a48dd45fe10a1b</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/Avatar-mount_base.html">893</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/Avatar-mount.html">903</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/Button-mount_base.html">545</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/Button-mount.html">550</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/FluentProvider-mount_base.html">1796</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/FluentProvider-mount.html">1767</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/FluentProviderWithTheme-mount_base.html">299</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/FluentProviderWithTheme-mount.html">286</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">221</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">230</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">315</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">300</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/MakeStyles-mount_base.html">1567</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23493/perf-test-react-components/MakeStyles-mount.html">1566</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupLayout.stories.tsx [hunk 1]
```diff
@@ -1,49 +1,54 @@
...
+        <AvatarGroupItem name="Wanda Howard" />
+        <AvatarGroupItem name="Mona Kane" />
+        <AvatarGroupItem name="Allan Munger" />
+        <AvatarGroupItem name="Daisy Phillips" />
+        <AvatarGroupItem name="Robert Tolbert" />
+        <AvatarGroupItem name="Kevin Sturgis" />
+      </AvatarGroup>
+    </div>
   );
 };
```
Comment (khmakoto) [inline packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupLayout.stories.tsx:54]: You should add a description to this story as well.

File: packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupMaxAvatars.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        <AvatarGroupItem name="Kevin Sturgis" />
+      </AvatarGroup>
+    </div>
+  );
+};
+
+MaxAvatars.parameters = {
+  docs: {
+    description: {
+      story: 'An AvatarGroup supports a custom number of AvatarGroupItems to render before overflowing.',
```
Comment (khmakoto) [inline packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupMaxAvatars.stories.tsx:48]: We should probably also add a note of how many is the max count by default.

File: packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupLayout.stories.tsx [hunk 3]
```diff
@@ -52,3 +52,11 @@ export const Layout = () => {
     </div>
   );
 };
+
+Layout.parameters = {
+  docs: {
+    description: {
+      story: `An AvatarGroup supports support three layouts spread, stack, and pie. The default is spread.`,
```
Comment (khmakoto) [inline packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupLayout.stories.tsx:59]:
```suggestion
      story: `An AvatarGroup supports support three layouts: spread, stack, and pie. The default is spread.`,
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**